### PR TITLE
Add a "completion" screen to the onboarding

### PR DESCRIPTION
**Pull request by thomasguillot**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-plugin/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

This PR updates a bunch of components and most importantly, add a completion screen at the end of the onboarding. This allows users to choose what they want to do once they are done instead of being redirected to the dashboard and not knowing what to do.

Screenshot 2021-12-17 at 17 30 40

I've also reorganised the welcome screen a bit so both screen kind of match

Screenshot 2021-12-17 at 17 30 25

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Go through the Setup

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

**Review by dkoo — CHANGES_REQUESTED** [review]
A couple of non-blocking code style suggestions, and one blocking bug that prevented me from actually completing setup.

File: assets/wizards/setup/views/completed/index.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+					<ActionCard
+						title={ __( 'Configure Newspack', 'newspack' ) }
+						description={ __(
+							'Go in-depth with our various options to set up Newspack to meet your needs.',
+							'newspack'
+						) }
+					>
+						<div className="flex flex-row-reverse">
+							<Button isPrimary isSmall href={ newspack_urls.dashboard }>
+								{ __( 'Go to Dashboard', 'newspack' ) }
```
[the remark this answers is not in the document]
Comment (dkoo) [reply assets/wizards/setup/views/completed/index.js:51]: This is the blocking bug that we need to fix before we can proceed.

File: assets/components/src/footer/index.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ import './style.scss';
 
 const Footer = ( { simple } ) => {
 	const componentsDemo = window && window.newspack_urls && window.newspack_urls.components_demo;
```
[the remark this answers is not in the document]
Comment (thomasguillot) [reply assets/components/src/footer/index.js]: https://github.com/Automattic/newspack-plugin/pull/1377/commits/412655c206415a39c2e06b3e30beb068d456a4e6

File: assets/wizards/setup/views/completed/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (thomasguillot) [reply assets/wizards/setup/views/completed/index.js:60]: I attempted to fix this here: https://github.com/Automattic/newspack-plugin/pull/1377/commits/d238f1fd680361a71169361f41fa419e3d40bc95

Not sure if this is ok. If it's not then I could use some help 🙂

[the remark this answers is not in the document]
Comment (dkoo) [reply assets/wizards/setup/views/completed/index.js:60]: @thomasguillot This seems to have worked when I tested on a new JN site. Thanks!

**Review by dkoo — APPROVED** [review]
@thomasguillot I updated the branch with `master` and fixed merge conflicts—could you do one more round of functional testing on your end to make sure the merge went okay, then if it looks 👍 to you I think this is good to go.

**Comment by thomasguillot** [comment]
@dkoo thank you so much for this. It looked good. I just added a very small fix to the mailchimp error message in https://github.com/Automattic/newspack-plugin/pull/1377/commits/1f11048792668eb04a1ab62a60513de34a58568d and https://github.com/Automattic/newspack-plugin/pull/1377/commits/dc1de65d2b09d63ebf22f8b727a44d9fa2bf52d9

**Pull request merged**